submissions/busolaAdedoyin/htmlForms/contactForm.html

```diff
@@ -5,9 +5,15 @@
     <title>Login Page</title>
```

Comment:
You would need to update the title of this page, the title differs from the contact.

Reply:
@elizabethbusola